### ISSUE TITLE
Wordpress 4.0 Stuck on No Update Required

### ISSUE THREAD

**Issue by vmetayer** [mchung/heroku-buildpack-wordpress#82]
Hi-

I set the WORDPRESS_VERSION config to 4.0.  It succesfully upgraded, I navigated to /wp-admin.  Wordpress requested a db update.  However, I cannot access my dashboard.  I am stuck on a page with this address site/wp-admin/upgrade.php?_wp_http_referer=%2Fwp-admin%2F

the message is No Update Required
Your WordPress database is already up-to-date!

![screen shot 2014-09-07 at 6 39 51 pm](https://cloud.githubusercontent.com/assets/789200/4179987/e9589310-36df-11e4-8331-7b75a89568b6.png)

Thank you

**Comment by nicdutil** [comment]
Same here.. Flushing the cache returns me back to an update to version 4.1 page.

Nicolas

**Comment by janharold** [comment]
Yeah. Same here. Looping back to 4.1 update and no update required when you do the memcache flush.
